packages/web-components/package.json

```diff
@@ -218,7 +218,8 @@
   "devDependencies": {
     "@types/web": "^0.0.142",
     "@storybook/html": "6.5.15",
-    "@tensile-perf/web-components": "~0.1.13"
+    "@tensile-perf/web-components": "~0.1.15",
```

Comment:
we should move this to root package.json to adhere to single version policy + other packages can use this in future/now.

also follow-up based on our offline convo. once chromium bumps we will need to do the bump manually every time both tensile and chromedriver.

Reply:
I reverted the move to root because it created a lot of odd breaks - we can take as a follow-up as you mentioned.